CHANGES.md

```diff
@@ -1,5 +1,26 @@
 # Changes
 
+## edge-20.6.3
+
+This edge release is our release candidate for 2.8.1. It includes improvements
+to the multicluster support functionality as well as other fixes.
+
+* Controller
+  * Added an option to change the set of networks for which the proxy
+    should do discovery by IP address
+  * Upgraded the version of Grafana to 7.0.3
+  * Improved multicluster support to be able to work with gateways that
+    expose a hostname rather than a concrete IP address
```

Comment:
In [2.8.0](https://github.com/linkerd/linkerd2/releases/tag/stable-2.8.0) notes we explicitly called out that multicluster does not work on EKS.

For users who are not aware *why* EKS was not supported (this reason), I think we should similarly call it out as supported now.

```suggestion
  * Added multicluster support for Amazon EKS by allowing gateways to expose a
    hostname rather than a concrete IP address
```

Or something like that!